packages/debug/src/browser/debug-preferences.ts

```diff
@@ -49,6 +49,11 @@ export const debugPreferencesSchema: PreferenceSchema = {
             enum: ['never', 'always', 'onFirstSessionStart'],
             description: 'Controls when the debug status bar should be visible.',
             default: 'onFirstSessionStart'
+        },
+        'debug.decorations.badges': {
+            type: 'boolean',
+            default: true,
+            description: 'Controls whether badges are enabled for debug.'
```

Comment:
nit: I think we might want to at least mention the actual view (decorations are for views themselves):

```suggestion
            description: 'Controls whether badges are enabled for the debug view.'
```